### PR TITLE
[Feat] Support Statically Exported Websites without next export 

### PR DESCRIPTION
**Pull request by iamvishnusankar**

### CODE REVIEW

File: examples/static-export/next-sitemap.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/** @type {import('next-sitemap').IConfig} */
+const config = {
+  siteUrl: process.env.SITE_URL || 'https://example.com',
+  generateRobotsTxt: true,
+  output: 'static', // Set static output here
```
Comment (ovflowd) [inline examples/static-export/next-sitemap.config.js:6]: Shouldn't this be "export"?

Comment (iamvishnusankar) [reply examples/static-export/next-sitemap.config.js]: @ovflowd Thanks for pointing it out, fixed in example config

Comment (ovflowd) [reply examples/static-export/next-sitemap.config.js]: You welcome :D

**Comment by iamvishnusankar** [comment]
OMG! Never thought this was meant to be part of the OG website! 🥳

**Comment by ovflowd** [comment]
Hehe, now you know :)

**Comment by ovflowd** [comment]
Off-topic: @iamvishnusankar when is the release of 4.1.0 supposed to take place? 👀

**Comment by iamvishnusankar** [comment]
@ovflowd `4.1.3` is live now!

**Pull request merged**